common/text/text_structure.h

```diff
@@ -95,10 +95,14 @@ class TextStructureView {
 
   const LineColumnMap& GetLineColumnMap() const { return line_column_map_; }
 
+  // Given a byte offset, return the line/column
   LineColumn GetLineColAtOffset(int bytes_offset) const {
     return line_column_map_.GetLineColAtOffset(contents_, bytes_offset);
   }
 
+  // Convenience function: Given the token, return the range it covers.
+  LineColumnRange GetRangeForToken(const TokenInfo& token) const;
```

Comment:
small request for a few unit-tests on this function

Reply:
Done.